css/style.css

```diff
@@ -0,0 +1,1140 @@
+html {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0 auto;
+  font-family: "Montserrat", sans-serif;
+  font-style: normal;
+  max-width: 1440px;
+}
+
+.header {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: space-between;
+  padding: 24px 60px 0 42.72px;
+  width: 100%;
+}
+
+.header__logo {
+  display: inline-flex;
+}
+
+.header__nav {
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: flex-end;
+  margin-left: 110px;
+}
+
+.header__right {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  align-items: flex-end;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  margin-right: 0;
+}
+
+.right__menu {
+  margin-right: -25px;
+}
+
+.nav__link {
+  white-space: nowrap;
+  font-weight: 600;
+  font-size: 15px;
+  color: #000;
+  text-decoration: none;
+  padding: 5px;
+  margin-right: 30px;
+  border-bottom: 2px solid transparent;
+}
+
+.burger {
+  border: none;
+  background: white;
+}
+
+@media (max-width: 1100px) {
+  .header__nav,
+  .right__menu {
+    display: none;
+  }
+}
+
+.nav__link:hover,
+.nav__link:focus,
+.nav__link:active {
+  color: #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla-Project_Evgen/pullRequest?prid=6071575)